google/cloud/firestore_v1/query.py

```diff
@@ -86,6 +85,8 @@ class Query(object):
             The "order by" entries to use in the query.
         limit (Optional[int]):
             The maximum number of documents the query is allowed to return.
+        limit_to_last (Optional[bool]):
+            Limit query results to last.
```

Comment:
to last what?

Reply:
Oh, looks like I've lost the think just in a middle of an action. Rewritten it in Java's comment fashion